packages/create-rsbuild/src/index.ts

```diff
@@ -56,6 +72,12 @@ export async function main() {
   const packageJsonPath = path.join(packageRoot, 'package.json');
   const { version } = require(packageJsonPath);
 
+  const currentPath = path.join(cwd);
+  // check cwd is empty
```

Comment:
I'm wondering why do we need to check `cwd` folder

Reply:
> I'm wondering why do we need to check `cwd` folder

When I develop scaffolding tools, I usually check if the current directory contains any other files. The scenario I have in mind is that users often want to create a project in a clean directory. This way, we can provide a friendly reminder to the user. Of course, this is not crucial, and you can decide whether to merge this PR.

Reply:
We have already checked the `target` folder, and all new files will be output to the `target` folder instead of `cwd`. So I don't understand why we need to check the `cwd` directory.